src/components/Todo.vue

```diff
@@ -10,22 +10,28 @@
 </template>
 
 <script>
-import { Vue, Component, Prop, p, Watch } from "av-ts"
+import * as Vue from "vue"
+import { Component, Prop, Watch } from "vue-property-decorator"
 
 @Component({
   name: "todo"
 })
 class Todo extends Vue {
-  @Prop todoId = p({ type: Number, required: true })
-  @Prop description = p({ type: String, required: true })
-  @Prop type = p({ type: String, required: true })
-  @Prop estimatedSize = p({ type: Number, required: true })
-  @Prop status = p({ type: Boolean, required: true })
-
   instanceStatus = this.status
 
+  @Prop({ required: true })
+  todoId: number
```

Comment:
To me, it's better than `av-ts` with below reason.

- it's more natural with typescript style
- there is no magic method like `p`, all magic happened in decorator.